plaso/parsers/manager.py

```diff
@@ -327,7 +189,9 @@ def GetParserObjects(cls, parser_filter_expression=None):
     Returns:
       dict[str, BaseParser]: parsers per name.
     """
-    includes, excludes = cls._GetParserFilters(parser_filter_expression)
+    parser_filter_helper = parser_filter.ParserFilterExpressionHelper(None)
```

Comment:
Please add something to the docstring mentioning that this expression can't contain presets names.

Reply:
ack